stdlib/importlib/metadata/__init__.pyi

```diff
@@ -35,6 +35,23 @@ if sys.version_info >= (3, 8):
             dist: ClassVar[Distribution | None]
             def matches(self, **params: Any) -> bool: ...  # undocumented
 
+    if sys.version_info >= (3, 10):
+        class EntryPoints(list[EntryPoint]):  # use as list is deprecated since 3.10
```

Comment:
We also need to override `__getitem__`. `list.__getitem__` accepts `SupportsIndex`, while this accepts `int` (deprecated) and `str`.

Reply:
Wasn't sure if I should include `__getitem__` in the stubs at first :) Will add it.

Reply:
Added.